index.bs

```diff
@@ -1405,7 +1405,7 @@ SessionResult = (StatusResult)
 <div algorithm>
 
 To <dfn>update the event map</dfn>, given
-|session|, |list of event names|, |browsing contexts|, and |enabled|:
+|session|, |list of requested event names|, |browsing contexts|, and |enabled|:
```

Comment:
Can we remove "list of" from the variable name i.e. just have `|requested event names|`